contracts/core/mpl-rewards/v1/test/helper/Rewards.sol

```diff
@@ -106,7 +106,7 @@ contract CustodialTestHelper is TestUtil {
             amt        *= WAD;
             custodyAmt *= WAD;
         } else {
-            // Transfer BPTs to the Farmers i.e Fay and Fez. Although both can use interchangeably but to make it code consistent chosses to use Fay anf Fez. 
+            // Transfer BPTs to the Farmers i.e Fay and Fez. Although both can use interchangeably but to make it code consistent chosses to use Fay and Fez. 
```

Comment:
Didn't realize this was the same comment as above, should be:
`// Transfer BPTs to the Farmers i.e Fay and Fez. Although both can use interchangeably but to make code consistent chose to use Fay and Fez.`